docs/releasenotes/release-1.118.md

```diff
@@ -1,26 +1,26 @@
-# v1.118.0
+# v1.118.1
 
-** This version is not yet released; this document is gathering release notes for the future release **
+* This release introduces a new mechanism to reconcile KCC resource. Currently it only applies to `LoggingLogMetric`.
```

Comment:
Maybe "This release introduces our new direct-reconciliation mechanism to reconcile KCC resources (without relying on terraform).  Currently ..."

Just because "new mechanism" may prompt more questions than it answers!

Reply:
Sg!